### DISCUSSION TITLE
⛔ 🛟 Support Request: When I want to edit, it slows down a little bit

### DISCUSSION THREAD

**Discussion by Soulgear99** [elementor/elementor#24317] [Developers Community]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

There's a slight delay when attempting to create or edit a page using Elementor.
[Load faster.webm](https://github.com/elementor/elementor/assets/70857278/105f82cf-588d-4340-bac5-92f1f56a7f15)

### Describe the solution you'd like

It would be better if it loaded faster when I want to edit.

**Comment by nicholaszein** [comment]
Hello @Soulgear99,

Thanks for contacting us! 🙏 

### 🛟 This is more of a support question, not really something we can help with here. 

👉 Please remember to check the [Help Center](https://elementor.com/help) or contact [Elementor Support](https://elementor.com/contact) before submitting an issue.


### ⚠️ Note that GitHub Discussions are reserved for Elementor 🚀 Feature Requests only, and should not be used for support.

❔ If you have questions, or need guidance, help, or general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) The main Facebook group where all kinds of users come together to help each other.
- [Elementor Academy](https://elementor.com/academy) Learn everything you need to know about using Elementor.
- [Elementor Developers Documentation](https://developers.elementor.com/) Learn everything about our Developer-API.
- [Free Support Forum](https://wordpress.org/support/plugin/elementor) Get help on WordPress.org.

💎 If you have an active [Elementor Pro](https://elementor.com/pro) license you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

### ⚠️ As a gentle reminder, **🛟 Support Request Discussions** may need to be closed without troubleshooting. 
Remember to always follow our **[🚀 Feature Request Guidelines](https://elemn.to/gh-new-feature-request)** to save time and ensure smooth troubleshooting.

Thank you for understanding! 🙏 

Kind regards
